src/components/WorkPoolFlowRunCompletePercentage.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <span v-if="completePercent" class="work-pool-flow-run-complete-percentage">
+    <span>{{ completePercent }}%</span>
+    <span v-if="percentChange" class="work-pool-flow-run-complete-percentage__difference" :class="percentChangeClasses">
+      {{ percentChange.direction }}
+      {{ percentChange.change }}
+    </span>
+  </span>
+  <span v-else class="work-pool-flow-run-complete-percentage__none">
+    N/A
+  </span>
+</template>
+
+<script lang="ts" setup>
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { useWorkspaceApi } from '@/compositions'
+  import { WorkPool, FlowRunsFilter } from '@/models'
+  import { mapper } from '@/services'
+  import { WorkspaceDashboardFilter } from '@/types'
+  import { toPercent } from '@/utilities'
+
+  const props = defineProps<{
+    workPool: WorkPool,
+    filter?: WorkspaceDashboardFilter,
+  }>()
+
+  const subscriptionOptions = {
+    interval: 30000,
+  }
```

Comment:
at some point we should probably make a const for the interval the dashboard uses. I haven't even added intervals to the components I've made yet.

Reply:
I'll do this in a separate PR and use it here before I merge.

Reply:
What's a good place to put something like that? Should I make a `useDashboardSubscriptionOptions` composition?

Reply:
Yes, that would give us a common place to do abstractions in the future